ewah/hooks/plentymarkets.py

```diff
@@ -220,12 +221,18 @@ def get_data_in_batches(
         url = self.endpoint + resource
 
         if data_from:
-            params[self._INCREMENTAL_FIELDS[resource][0]] = data_from.isoformat()
+            if request_method == "get":
+                params[self._INCREMENTAL_FIELDS[resource][0]] = data_from.isoformat()
+            elif request_method == "post":
+                post_request_payload[
+                    self._INCREMENTAL_FIELDS[resource][0]
+                ] = data_from.isoformat()
         if data_until:
-            if resource == "/rest/accounts/contacts":
-                # inconsistent API implementation - ignores data for last day otherwise
-                data_until += timedelta(days=1)
-            params[self._INCREMENTAL_FIELDS[resource][1]] = data_until.isoformat()
+            if request_method == "get":
```

Comment:
Request (blocking): Can you also add an `elif` for `post`?

Reply:
Ah, I should probably read that first.

Could you still add an `elif`, add another `if` to check for the specific endpoint, and add a comment why you're doing what you're doing? Otherwise, we'll lose this info :)

Reply:
Added the elif. No need for the comment - will do that in the specific Plentymarkets DAG in the data loading repo